src/xsdba/utils.py

```diff
@@ -487,23 +520,23 @@ def rank(
 
     Parameters
     ----------
-    da : xr.DataArray
-        Source array.
+    da: xr.DataArray
+      Source array.
     dim : str | list[str], hashable
-        Dimension(s) over which to compute rank.
+      Dimension(s) over which to compute rank.
     pct : bool, optional
-        If True, compute percentage ranks, otherwise compute integer ranks.
-        Percentage ranks range from 0 to 1, in opposition to xarray's implementation,
-        where they range from 1/N to 1.
+      If True, compute percentage ranks, otherwise compute integer ranks.
+      Percentage ranks range from 0 to 1, in opposition to xarray's implementation,
+      where they range from 1/N to 1.
```

Comment:
The spacing changes here are a regression.

Reply:
is there a package to format this automatically?

Reply:
Not to my knowledge, unfortunately.